submissions/Maksym-Perepelytsia/html-css-popup/style.css

```
@@ -0,0 +1,204 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    text-decoration: none;
+}
+
+body {
+    font-family: Tahoma, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.5em;
+    background: #fff;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+}
+
+.header,
+.footer {
+    display: flex;
+    align-items: center;
+    background: #D3D3D3;
+    box-shadow: 0px 0px 3px 0px rgba(0, 0, 0, 0.5);
+}
+
+.header,
+.main,
+.footer{
+    padding: 15px;
+}
+
+.header {
+    justify-content: space-between;
+}
+
+.header__logo img,
+.popup-label img {
+    display:block;
+}
+
+.header__menu {
+    display: flex;
+}
+
+.header__menu__item .menu__link,
+.header__menu__item .popup-label{
+    padding:5px;
+    margin:5px;
+}
+
+.menu__link {
+    display: block;
+    color: #000;
+    position: relative;
+}
+
+.menu__link:after {
+    content: "";
+    position: absolute;
+    left:0;
+    bottom: 5px;
+    display: block;
+    opacity: 0;
+    width:100%;
+    height: 1px;
+    background: transparent;
+    transition: 0.5s;
+    transition-timing-function: cubic-bezier(.58,.3,.005,1);
+}
+
+.menu__link:hover:after {
+    opacity: 1;
+    background: linear-gradient(90deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 50%, rgba(0,0,0,0) 100%);
+}   
+
+.popup-toggle {
+    border-radius: 50%;
+}
+
+#popup-checkbox,
+#more-cats-checkbox{
+    appearance: none;
+    position:absolute;
```

```suggestion
    position: absolute;
```
You forget space here